gocourse03/micemetropolis/main.go

```diff
@@ -0,0 +1,157 @@
+package micemetropolis
+
+import (
+	"fmt"
+	"math/rand"
+	"time"
+)
+
+type (
+	Sector     string
+	rodentType string
+	FromTo     [2]Sector
+
+	Movement struct {
+		Time time.Time
+		FromTo
+		rodentID int
+	}
+
+	Rodent struct {
+		ID        int
+		Type      rodentType
+		Sector    Sector
+		UpdatedAt time.Time
+	}
+)
+
+func (m *Movement) Show() {
+	fmt.Print(m.rodentID, " ", m.FromTo[0], "->", m.FromTo[1], " ", m.Time.Format("2006-01-02 15:04:05"), "\n")
```

Comment:
Можна простіше:
```suggestion
    fmt.Printf("%s %s->%s %s\n", m.rodentID, m.FromTo[0], m.FromTo[1], m.Time.Format(time.DateTime))
}
```

Зазвичай використовують `Printf`, якщо треба якесь форматування для кількох значень.